js/core/index.js

```diff
@@ -24,6 +24,15 @@ var keyboard = require('./keyboard');
 var ps2 = require('./ps2');
 var pci = require('./pci');
 var net = require('./net');
+var stdio = require('./stdio');
+
+global.stdout = new stdio.stdout();
+global.stdin = new stdio.stdin();
```

Comment:
I think it's nice that they're separate though, we could pass them as an environment argument to commands `env = { stdin, stdout, ...other env stuff }`